cpp/pipelines/collect_journal_stats.cc

```diff
@@ -0,0 +1,213 @@
+        std::string volume;
+        std::string issue(_936_field->getFirstSubfieldWithCode('e'));
+        if (issue.empty())
+            issue = _936_field->getFirstSubfieldWithCode('d');
+        else
+            volume = _936_field->getFirstSubfieldWithCode('d');
+        const std::string year(_936_field->getFirstSubfieldWithCode('j'));
+
+        std::map<std::string, std::string> column_names_to_values_map{
+            { "timestamp",     JOB_START_TIME                                                         },
+            { "Quellrechner",  HOSTNAME                                                               },
+            { "Systemtyp",     system_type,                                                           },
+            { "Zeder_ID",      std::to_string(zeder_id_and_type->second.zeder_id_)                    },
+            { "Zeder_URL",     ZEDER_URL_PREFIX + std::to_string(zeder_id_and_type->second.zeder_id_) },
+            { "PPN_Typ",       std::string(1, zeder_id_and_type->second.type_)                        },
+            { "PPN",           superior_control_number                                                },
+            { "Jahr",          std::to_string(YearStringToShort(year))                                },
+            { "Band",          volume                                                                 },
+            { "Heft",          issue                                                                  },
+            { "Seitenbereich", pages                                                                  }
+        };
+
+        unsigned start_page, end_page;
+        if (SplitPageNumbers(pages, &start_page, &end_page)) {
+            column_names_to_values_map["Startseite"] = StringUtil::ToString(start_page);
+            column_names_to_values_map["Endseite"]   = StringUtil::ToString(end_page);
+        }
+
+        db_connection->insertIntoTableOrDie("zeder.erschliessung", column_names_to_values_map,
+                                            DbConnection::DKB_REPLACE, "PPN='" + superior_control_number + "'");
+        ++inserted_count;
+    }
+
+    LOG_INFO("Processed " + std::to_string(total_count) + " records and inserted " + std::to_string(inserted_count)
+             + " into Ingo's database.");
+}
+
+
+} // unnamed namespace
+
+
+int Main(int argc, char *argv[]) {
+    if (argc != 3)
+        ::Usage("[--min-log-level=log_level] system_type marc_titles_records\n"
+                "\twhere \"system_type\" must be one of ixtheo|krimdok");
+
+    std::string system_type;
+    if (__builtin_strcmp("ixtheo", argv[1]))
+        system_type = "ixtheo";
+    else if (__builtin_strcmp("krimdok", argv[1]))
+        system_type = "krimdok";
+    else
+        LOG_ERROR("system_type must be one of ixtheo|krimdok|relbib!");
+
+    const auto ppns_to_zeder_ids_and_types_map(GetPPNsToZederIdsAndTypesMap(system_type));
+
+    IniFile ini_file;
```

Comment:
INI path missing (which INI file contains Zeder MySQL credentials?)

Reply:
No, the path is derived from the executable name and the file resides in the standard location.